pleskdistup/common/src/util.py

```diff
@@ -9,31 +9,63 @@
 # Returns standard output
 def logged_check_call(cmd: typing.Union[typing.Sequence[str], str], **kwargs) -> str:
     log.info(f"Running: {cmd!r}. Output:")
+    stdout = []
+
+    def proc_stdout(line: str) -> None:
+        log.info(line, to_stream=False)
+        stdout.append(line)
+
+    def proc_stderr(line: str) -> None:
+        log.err(line)
+
+    exit_code = exec_get_output_streamed(cmd, proc_stdout, proc_stderr, **kwargs)
+    if exit_code != 0:
+        log.err(f"Command {cmd!r} failed with return code {exit_code}")
+        raise subprocess.CalledProcessError(returncode=exit_code, cmd=cmd, output="\n".join(stdout))
+
+    log.info(f"Command {cmd!r} finished successfully")
+    return "\n".join(stdout)
 
-    # I beleive we should be able pass argument to the subprocess function
+
+# Allows to get stdout/stderr by streaming line by line, calling callbacks
+# and returns process exit code
```

Comment:
There are function documentation strings for that.

Reply:
Removed these lines

Reply:
Right. Let's not document anything because otherwise we **need** to document arguments. (Actually we don't.)

Reply:
Added. But we better impose similar conditions on other functions or have a consistent approach. Then, any programmer, by default, would add similar docstrings (at least).